sympy/core/numbers.py

```diff
@@ -2002,11 +2002,13 @@ def __new__(cls, expr, coeffs=Tuple(), alias=None, **args):
 
         else:
             rep = DMP.from_list([1, 0], 0, dom)
+            scoeffs = Tuple(1, 0)
 
             if root.is_negative:
                 rep = -rep
+                scoeffs = Tuple(-1, 0)
 
-            sargs = (root, coeffs)
+            sargs = (root, scoeffs)
 
```

Comment:
Yes. Besides, I suspect that the else part will have to be dropped unless someone finds an explanation for the sign change.